Pipes that don't have a pipeline throw NPE in configure

**Describe the issue**
Caused by: java.lang.NullPointerException
	at nl.nn.adapterframework.pipes.AbstractPipe.configure(AbstractPipe.java:174) ~[ibis-adapterframework-core-7.6.1-20210922.162808.jar:7.6.1-20210922.162808]
	at nl.nn.adapterframework.pipes.FixedForwardPipe.configure(FixedForwardPipe.java:57) ~[ibis-adapterframework-core-7.6.1-20210922.162808.jar:7.6.1-20210922.162808]
	at nl.nn.adapterframework.soap.SoapWrapperPipe.configure(SoapWrapperPipe.java:116) ~[ibis-adapterframework-core-7.6.1-20210922.162808.jar:7.6.1-20210922.162808]
	at nl.nn.adapterframework.extensions.esb.EsbSoapWrapperPipe.configure(EsbSoapWrapperPipe.java:331) ~[ibis-adapterframework-core-7.6.1-20210922.162808.jar:7.6.1-20210922.162808]
	at nl.nn.adapterframework.extensions.esb.DirectWrapperPipe.doPipeWithTimeoutGuarded(DirectWrapperPipe.java:74) ~[ibis-adapterframework-core-7.6.1-20210922.162808.jar:7.6.1-20210922.162808]
	... 109 more

**Reporter**
Ap/Ali

--- a/core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java
+++ b/core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java
@@ -164,7 +164,7 @@ public abstract class AbstractPipe extends TransactionAttributes implements IExt
 		if (pipeForwards.isEmpty()) {
 			//TODO pipe will follow the next forward no need to show warning
 			ConfigurationWarnings.add(this, log, "has no pipe forwards defined");
-		} else {
+		} else if (pipeline != null){
 			for (Iterator<String> it = pipeForwards.keySet().iterator(); it.hasNext();) {
 				String forwardName = it.next();
 				PipeForward forward= pipeForwards.get(forwardName);

